<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="mainLayout">
    <item>
     <widget class="QSplitter" name="horizontalSplitter">
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
      <widget class="QSplitter" name="splitter">
       <property name="orientation">
        <enum>Qt::Orientation::Vertical</enum>
       </property>
       <widget class="QFrame" name="FFT_frame">
        <layout class="QGridLayout" name="gridLayoutFFT">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <item row="1" column="1">
          <widget class="QwtPlot" name="FFT_plot" native="true"/>
         </item>
         <item row="1" column="0">
          <widget class="QWidget" name="fftYAxisContainer">
           <layout class="QVBoxLayout" name="fftYLayout">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="contentsMargins">
             <left>0</left>
             <top>0</top>
             <right>0</right>
             <bottom>0</bottom>
            </property>
            <item>
             <widget class="QToolButton" name="fftPlusY">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="fftYTitle">
              <property name="text">
               <string>Log Magnitude</string>
              </property>
              <property name="rotationAngle">
               <number>270</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="fftMinusY">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QWidget" name="fftXAxisContainer">
           <layout class="QHBoxLayout" name="fftXLayout">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="contentsMargins">
             <left>0</left>
             <top>0</top>
             <right>0</right>
             <bottom>0</bottom>
            </property>
            <item>
             <widget class="QToolButton" name="fftMinusX">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignVCenter">
             <widget class="QLabel" name="fftXTitle">
              <property name="text">
               <string>Frequency</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="fftPlusX">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QFrame" name="Time_frame">
        <layout class="QGridLayout" name="gridLayoutTime">
         <property name="horizontalSpacing">
          <number>2</number>
         </property>
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <item row="1" column="1">
          <widget class="QwtPlot" name="Time_plot" native="true"/>
         </item>
         <item row="1" column="0">
          <widget class="QWidget" name="timeYAxisContainer">
           <layout class="QVBoxLayout" name="timeYLayout">
            <property name="spacing">
             <number>1</number>
            </property>
            <property name="contentsMargins">
             <left>0</left>
             <top>0</top>
             <right>0</right>
             <bottom>0</bottom>
            </property>
            <item>
             <widget class="QToolButton" name="timePlusY">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QLabel" name="timeYTitle">
              <property name="text">
               <string>Power (µW)</string>
              </property>
              <property name="rotationAngle">
               <number>270</number>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="timeMinusY">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QWidget" name="timeXAxisContainer">
           <layout class="QHBoxLayout" name="timeXLayout">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="contentsMargins">
             <left>0</left>
             <top>0</top>
             <right>0</right>
             <bottom>0</bottom>
            </property>
            <item>
             <widget class="QToolButton" name="timeMinusX">
              <property name="text">
               <string>-</string>
              </property>
             </widget>
            </item>
            <item alignment="Qt::AlignVCenter">
             <widget class="QLabel" name="timeXTitle">
              <property name="text">
               <string>Time (µs)</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="timePlusX">
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
      <widget class="QWidget" name="sidePanel">
       <layout class="QVBoxLayout" name="sideLayout">
        <item>
         <widget class="QComboBox" name="modes">
          <item>
           <property name="text">
            <string>0-40MHz</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>0-100kHz</string>
           </property>
          </item>
         </widget>
        </item>
        <item alignment="Qt::AlignmentFlag::AlignHCenter|Qt::AlignmentFlag::AlignVCenter">
         <widget class="QLabel" name="PeakFreq">
          <property name="text">
           <string>Peak Freq</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="spacerTop">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="PausePlay"/>
        </item>
        <item>
         <widget class="QPushButton" name="Save"/>
        </item>
        <item>
         <spacer name="spacerBottom">
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="Logo">
    <property name="title">
     <string>Ultracoustics</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuLicensing">
    <property name="title">
     <string>Licensing</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <addaction name="Logo"/>
   <addaction name="menuLicensing"/>
   <addaction name="menuView"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwtPlot</class>
   <extends>QWidget</extends>
   <header>qwt_plot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
